server/src/main/java/org/opentosca/toscana/plugins/kubernetes/visitor/KubernetesNodeVisitor.java

```diff
@@ -0,0 +1,57 @@
+package org.opentosca.toscana.plugins.kubernetes.visitor;
+
+import java.util.Arrays;
+import java.util.Set;
+
+import org.opentosca.toscana.model.capability.Capability;
+import org.opentosca.toscana.model.capability.OsCapability;
+import org.opentosca.toscana.model.node.Compute;
+import org.opentosca.toscana.model.node.ContainerRuntime;
+import org.opentosca.toscana.model.node.DockerApplication;
+import org.opentosca.toscana.model.node.RootNode;
+import org.opentosca.toscana.model.visitor.NodeVisitor;
+import org.opentosca.toscana.plugins.kubernetes.visitor.policies.OsPolicy;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KubernetesNodeVisitor implements NodeVisitor {
+    private final static Logger logger = LoggerFactory.getLogger(KubernetesNodeVisitor.class.getName());
```

Comment:
Please use loggers supplied by transformationContext.getLogger(). This way api clients will also receive the logs.